test/Elastica/Query/IdsTest.php

```diff
@@ -176,27 +156,11 @@ public function testSetTypeSingleSearchArrayDocInOtherType()
     /**
      * @group functional
      */
-    public function testSetTypeAndAddType()
-    {
-        $this->markTestSkipped('ES6 update: the final mapping would have more than 1 type');
-
-        $query = new Ids();
-
-        $query->setIds(['1', '4']);
-        $query->setType('helloworld1');
-        $query->addType('helloworld2');
-
-        $resultSet = $this->_index->search($query);
-
-        $this->assertEquals(2, $resultSet->count());
-    }
-
-    /**
-     * @group functional
-     */
     public function testSetTypeArraySearchArray()
     {
-        $this->markTestSkipped('ES6 update: the final mapping would have more than 1 type');
+        if (!$this->_getVersion() < 6) {
```

Comment:
We should leave the code in, but should we leave the tests in?

Reply:
I agree... better to leave the code but remove the tests